hongit-api/src/main/kotlin/com/devfloor/hongit/api/article/domain/ArticleRepositoryCustomImpl.kt

```diff
@@ -29,4 +31,27 @@ class ArticleRepositoryCustomImpl(
             .limit(5)
             .fetch()
     }
+
+    override fun findAllByBoardOrderByViewCount(board: Board, pageRequest: PageRequest): List<Article> {
+        return jpaQueryFactory.selectFrom(article)
+            .join(articleViewCount).on(article.eq(articleViewCount.article))
+            .where(article.board.eq(board))
+            .orderBy(articleViewCount.count.desc())
```

Comment:
```suggestion
    override fun findAllByBoardOrderByViewCount(board: Board, pageRequest: PageRequest): List<Article> {
        return jpaQueryFactory
            .selectFrom(article)
            .join(articleViewCount).on(article.eq(articleViewCount.article))
            .where(article.board.eq(board))
            .orderBy(articleViewCount.count.desc())
```

여기 클래스 코드들에 select 부분 한칸 내리는건 어때? 그럼 좀더 코드를 가로로 읽을 필요 없이 세로로 읽어내려가기에 한번에 잘 보이는 것 같아서!!